src/SystemTextJson.JsonDiffPatch/SystemTextJson.JsonDiffPatch.csproj

```diff
@@ -11,11 +11,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'net461'" />
+    <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'net462'" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Text.Json" Version="6.0.0" />
+    <PackageReference Include="System.Text.Json" Version="7.0.3" />
```

Comment:
I'm not sure if we should go to the latest version. I reckon the base version `7.0.0` would be enough for most users and leave it to them to use patched versions? Thoughts?